package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Material style rating",
   "main": "lib/index.js",
   "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 0",
```

Comment:
We usually use [Standard JS](https://standardjs.com/) as a linter, so if you want, you can add `standard` as `test` command and add `standard` as dev dependency. That would make travis at least check if the code looks nice. :nail_care:

Reply:
(literally) :smile:

Reply:
Done